frontend/exporter/src/index_vec.rs

```diff
@@ -42,8 +44,14 @@ impl<I: Idx, T> Into<IndexVec<I, T>> for rustc_index::IndexVec<I, T> {
     }
 }
 
-impl<S, J: Idx, I: Idx + SInto<S, J>, U: Clone /*TODO: remove me?*/, T: SInto<S, U>>
-    SInto<S, IndexVec<J, U>> for IndexSlice<I, T>
+#[cfg(feature = "rustc")]
+impl<
+        S,
+        J: rustc_index::Idx,
+        I: rustc_index::Idx + SInto<S, J>,
+        U: Clone, /*TODO: remove me?*/
```

Comment:
What's up with this todo? Is there an issue for it?

Reply:
Good question, let me investigate.

Reply:
The bound  `Clone` is useful